sklearn/svm/classes.py

```diff
@@ -950,6 +956,15 @@ def __init__(self, kernel='rbf', degree=3, gamma='scale',
             shrinking=shrinking, probability=False, cache_size=cache_size,
             class_weight=None, max_iter=max_iter, random_state=None)
 
+    @deprecated("The ``probA_`` attribute will be deprecated ")
```

Comment:
For this PR, lets remove this deprecation and keep the update to the docstring regarding `fit_status_` and `shape_fit_`

Reply:
ok closing this. that pr is here: https://github.com/scikit-learn/scikit-learn/pull/14773